egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -305,7 +331,7 @@ def ParseSpliceString(splice_indexes):
     if (num_hidden_layers < args.num_lstm_layers):
         sys.exit("--num-lstm-layers : number of lstm layers has to be greater than number of layers, decided based on splice-indexes")
 
-    left_context = left_context + args.bptt_truncation_width + args.context_sensitive_chunk_width
+    left_context = left_context + args.chunk_left_context
```

Comment:
OK.  I prefer to do the addition left_context = left_context +
args.chunk_left_context
in the lstm training script rather than that python script, to keep that
python script's job simple. Dan


On Tue, Sep 1, 2015 at 5:06 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/lstm/make_configs.py
> <https://github.com/kaldi-asr/kaldi/pull/103#discussion_r38472127>:
>
> > @@ -305,7 +331,7 @@ def ParseSpliceString(splice_indexes):
> >      if (num_hidden_layers < args.num_lstm_layers):
> >          sys.exit("--num-lstm-layers : number of lstm layers has to be greater than number of layers, decided based on splice-indexes")
> >
> > -    left_context = left_context + args.bptt_truncation_width + args.context_sensitive_chunk_width
> > +    left_context = left_context + args.chunk_left_context
>
> It had to be provided to the make_config.py script as the same script also
> dumps the variables necessary for get_egs.sh. I will make necessary changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/103/files#r38472127>.
>